packages/react-components/src/components/Attachment/AttachmentUploadCards.tsx

```diff
@@ -75,7 +75,7 @@ export const _AttachmentUploadCards = (props: AttachmentUploadCardsProps): JSX.E
                 {
                   name: props.strings?.removeAttachment ?? 'Remove',
                   icon: (
-                    <div aria-label={removeAttachmentButtonString()}>
+                    <div aria-label={removeAttachmentButtonString()} data-ui-id="attachment-upload-card-remove">
```

Comment:
If it's needed for tests, please use `data-testid` instead 
Please see playwright and testing-library links for more details
https://playwright.dev/docs/locators#locate-by-test-id
https://testing-library.com/docs/queries/bytestid/